babel/messages/frontend.py

```diff
@@ -277,7 +277,7 @@ class extract_messages(Command):
          'path to the mapping configuration file'),
         ('no-location', None,
```

Comment:
because there are different types of options: `no-location` is [boolean option](https://github.com/python-babel/babel/blob/master/babel/messages/frontend.py#L885), `add-location` is [multiple value option](https://github.com/python-babel/babel/blob/master/babel/messages/frontend.py#L887)

example which does't work without this fix:
```
python setup.py extract_messages \
--mapping-file international/babel_mapping.ini \
--output-file proj/i18n/file.pot \
--charset utf-8 --add-location file \
--omit-header --sort-output
```
`omit-header` and `sort-output` -- boolean options, `output-file` and `add-location` -- multiple value options